### ISSUE TITLE
Implement PDF, HTML and image render functionality in `@solardoc/asciidoc-renderer` library

### ISSUE THREAD

**Issue by LisaPich** [SYP-AHIF-2023-24-25/SolarDoc#45]
Implement PDF, HTML and image render functionality in `@solardoc/asciidoc-renderer` library.

Depends on #48

**Comment by Luna-Klatzer** [comment]
Critical implementation issue due to:

https://github.com/asciidoctor/asciidoctor-reveal.js/issues/522

Development halted for now.

**Comment by Luna-Klatzer** [comment]
As this issue has become too complex to implement on its own, this issue will be closed and superseded by the following issues:
- #59
- #60
- #61
- #62
